code/modules/clothing/glasses/hud.dm

```diff
@@ -103,3 +103,9 @@
 	invisa_view = 2
 	toggleable = FALSE
 	actions_types = list()
+
+/obj/item/clothing/glasses/hud/health/sunmedhud
```

Comment:
the typepath can probably just be health/sunglasses, says it much more succinctly